doc/apiref.md

```diff
@@ -113,7 +113,7 @@ Sets permissions and ownership on a remote file.
     set_perms_on_remote(master, get_site_pp_path(master), '644', :owner => 'root', :group => 'root')
     ```
     
-### `inject_site_pp`
+### `inject_site_pp(host, path, manifest)`
```

Comment:
The options hash is only documented in the examples. It would be nice to have a full description of what options are available.